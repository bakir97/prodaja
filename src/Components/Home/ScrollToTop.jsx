import React from "react";
import { IconButton } from "@material-ui/core";
import NavigationIcon from "@material-ui/icons/Navigation";
const style = {
  button: { position: "fixed", bottom: "2rem", right: "1rem" },
  navigation: { fontSize: 30 }
};
const SCrollToTop = ({ scrollToTop }) => {
  return (
    <IconButton onClick={() => scrollToTop()} style={style.button}>
      <NavigationIcon style={style.navigation} />
    </IconButton>
  );
};

export default SCrollToTop;
