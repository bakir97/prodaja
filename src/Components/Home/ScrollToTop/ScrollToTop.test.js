import ScrollToTop from "./index";
describe("ScrollToTop", () => {
  const scrollToTop = jest.fn();
  const wrapper = shallow(<ScrollToTop scrollToTop={scrollToTop} />);
  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should call scrollToTop()", () => {
    wrapper.find("#button").simulate("click");
    expect(scrollToTop).toBeCalled();
  });
});
