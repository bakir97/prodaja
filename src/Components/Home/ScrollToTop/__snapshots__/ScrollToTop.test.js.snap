// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop should render 1`] = `
<WithStyles(IconButton)
  id="button"
  onClick={[Function]}
  style={
    Object {
      "bottom": "2rem",
      "position": "fixed",
      "right": "1rem",
    }
  }
>
  <pure(NavigationIcon)
    style={
      Object {
        "fontSize": 30,
      }
    }
  />
</WithStyles(IconButton)>
`;
